tour_model_eval/viz_similarity_unlabeled.ipynb

```diff
@@ -151,19 +153,57 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "great-leeds",
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "graph(sim)"
+    "graph(sim)\n"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "preceding-executive",
+   "id": "qualified-myrtle",
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "fig=plt.figure(1)\n",
+    "n = len(long_term_uuid_list)\n",
+    "for i in range(n):\n",
+    "    plt.subplot(n,n,i+1)\n",
+    "    data = eamtc.read_data(long_term_uuid_list[i])\n",
+    "    radius = 300\n",
```

Comment:
why is this in the for loop if it never changes?

Reply:
Do you mean `radius = 300\n"`? I will move it out. But the rest, I think I need to process the data from a user.

Reply:
yes. In general, my comment will be about the line just above my comment.